### ISSUE TITLE
Include option to define different test splits for pre-training and fine-tuning in transfer learning pipeline

### ISSUE THREAD

**Issue by JulianZabbarov** [DILiS-lab/SimbaML#28]
In some scenarios, the user might want to choose a larger test set when fine-tuning on the real-world dataset than when pre-training on the synthetic dataset. Thus it is useful to include the option to set to different test_splits for pre-training and fine-tuning, respectively.
